Library/Homebrew/formula.rb

```diff
@@ -2005,6 +2005,56 @@ def to_hash
     hsh
   end
 
+  # @private
+  def to_hash
+    hash = to_hash_without_variations
+    variations = {}
+
+    hash_keys_to_skip = %w[installed linked_keg pinned outdated]
```

Comment:
Why skip these? A comment might be nice.
```suggestion
    hash_keys_to_skip = %w[installed linked_keg pinned outdated].freeze
```

Reply:
Ah, actually I don't think we need these for formulae. Since casks could have different versions for different configurations, sometimes the `installed` and `outdated` keys would end up being different and we didn't want to overwrite any of these fields with info about whatever happened to be installed on the system that generated the JSON files.

For formulae, though, since versions can't differ, I don't think any of these would end up in the variations hash, anyway, so this can be removed.